app/services/data_prep_service.rb

```diff
@@ -0,0 +1,33 @@
+class DataPrepService
```

Comment:
I really like this poro! class methods on the outside, instance methods on the inside